packages/headless/src/api/search/search/product-recommendation.ts

```diff
@@ -1,70 +1,142 @@
-export interface ProductRecommendation {
+export interface DeprecatedProductRecommendation {
+  /**
+   * @deprecated Use `permanentid` instead.
+   */
+  sku?: string;
+  /**
+   * @deprecated Use `ec_name` instead.
+   */
+  name?: string;
+  /**
+   * @deprecated Use `clickUri` instead.
+   */
+  link?: string;
+  /**
+   * @deprecated Use `ec_brand` instead.
+   */
+  brand?: string;
+  /**
+   * @deprecated Use `ec_category` instead.
+   */
+  category?: string;
+  /**
+   * @deprecated Use `ec_price` instead.
+   */
+  price?: number;
+  /**
+   * @deprecated Use `ec_shortdesc` instead.
+   */
+  shortDescription?: string;
+  /**
+   * @deprecated Use `ec_thumbnails` instead.
+   */
+  thumbnailUrl?: string;
+  /**
+   * @deprecated Use `ec_images` instead.
+   */
+  imageUrls?: string[];
+  /**
+   * @deprecated Use `ec_promo_price` instead.
+   */
+  promoPrice?: number;
+  /**
+   * @deprecated Use `ec_in_stock` instead.
+   */
+  inStock?: boolean;
+  /**
+   * @deprecated Use `ec_rating` instead.
+   */
+  rating?: number;
+}
+
+export interface ProductRecommendation extends DeprecatedProductRecommendation {
```

Comment:
Looks like this change failed the build :thinking: 

Should I remove the `extends`?

Reply:
Yep, to properly resolve a publicly documented interface, the interfaces it is extending must be exported.

Personally, I think it is better to place everything in one interface rather than export `DeprecatedProductRecommendation` as part of our public contract.

Reply:
Alright, I put back everything in one interface